src/test/Makefile.inc

```diff
@@ -600,8 +600,9 @@ clean:
 	$(RM) *.o */*.o core *.core a.out *.log testfile* $(SYNC_FILE) $(TMP_HEADERS)
 
 clobber: clean
-	$(RM) $(TARGET) $(TARGET_STATIC_DEBUG) $(TARGET_STATIC_NONDEBUG)
+	$(RM) $(TARGET) $(TARGET).static-debug $(TARGET).static-nondebug
```

Comment:
isn't the problem here that `$(TARGET_STATIC_DEBUG)` and `$(TARGET_STATIC_NONDEBUG)` doesn't point at the right file?